src/components/sbb-tooltip/sbb-tooltip.scss

```diff
@@ -64,10 +64,7 @@
 
   position: var(--sbb-tooltip-position);
   pointer-events: all;
-  top: var(--sbb-tooltip-position-y);
-  left: var(--sbb-tooltip-position-x);
-  right: unset;
-  bottom: unset;
+  inset: var(--sbb-tooltip-position-y) unset unset var(--sbb-tooltip-position-x);
```

Comment:
Unfortunately the `unset` value is not supported in the inset shorthand, I've tried it in the past and found out that, for some strange reason, it doesn't work.

What we can do is to revert it or try to write it as `inset: var(--sbb-tooltip-position-y) auto auto var(--sbb-tooltip-position-x);` which should work.